test/it/uk/gov/hmrc/exports/scheduler/jobs/PurgeDraftDeclarationsJobISpec.scala

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmrc.exports.scheduler.jobs
+
+import testdata.ExportsTestData.eori
+import uk.gov.hmrc.exports.base.IntegrationTestPurgeSubmissionsToolSpec
+import uk.gov.hmrc.exports.models.declaration.DeclarationStatus._
+import uk.gov.hmrc.exports.models.declaration.ExportsDeclaration
+import uk.gov.hmrc.exports.scheduler.jobs.PurgeDraftDeclarationsJobISpec.uuids
+import uk.gov.hmrc.exports.util.{ExportsDeclarationBuilder, TimeUtils}
+
+import java.time.temporal.ChronoUnit
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class PurgeDraftDeclarationsJobISpec extends IntegrationTestPurgeSubmissionsToolSpec with ExportsDeclarationBuilder {
+
+  private val testJob: PurgeDraftDeclarationsJob = instanceOf[PurgeDraftDeclarationsJob]
+
+  private val beyondExpiryDateTime = TimeUtils.now().truncatedTo(ChronoUnit.MILLIS).minusSeconds(testJob.expireDuration.toSeconds).toInstant
+  private val recentlyUpdatedDateTime = TimeUtils.now().minusDays(1).toInstant
+
+  "PurgeDraftDeclarationsJob" should {
+
+    "remove DRAFT and AMENDMENT_DRAFT records" when {
```

Comment:
Nit: Can you stick some decs in here that are neither DRAFT or AMENDMENT_DRAFT but that are older than 30 days (to prove they don't get deleteded). :slightly_smiling_face:

Reply:
Yep!